src/quo/components/profile/expanded_collectible/view.cljs

```diff
@@ -0,0 +1,54 @@
+(ns quo.components.profile.expanded-collectible.view
+  (:require
+    [promesa.core :as p]
+    [quo.components.counter.collectible-counter.view :as collectible-counter]
+    [quo.components.profile.expanded-collectible.style :as style]
+    [quo.foundations.shadows :as shadows]
+    [quo.theme]
+    [react-native.core :as rn]
+    [schema.core :as schema]))
+
+(defn view-internal
+  [{:keys [container-style square? on-press counter image-src]}]
+  (let [[image-size set-image-size] (rn/use-state {})]
+    (rn/use-effect
+     (fn []
+       (p/let [[image-width image-height] (rn/image-get-size image-src)]
+         (set-image-size {:width        image-width
+                          :height       image-height
+                          :aspect-ratio (/ image-width image-height)})))
+     [image-src])
+    ^{:key (str "expanded-collectible-" image-src)}
```

Comment:
yeah I can see about removing this now. It was previously needed so that the dimensions of the image would update if the image-src changes. However I think my current approach does this anyway. 
Thanks for the reminder 👍

Reply:
removed, thanks!